src/node/rpc/member_frontend.h

```diff
@@ -946,6 +946,24 @@ namespace ccf
         .set_openapi_hidden(true)
         .install();
 
+      using JWTKeyMap = std::map<JwtKeyId, Cert>;
+
+      auto get_jwt_keys = [this](auto& ctx, nlohmann::json&& body) {
```

Comment:
We should also use this new endpoint in `jwt_issuer.py:wait_for_refresh`

Reply:
I'd missed, but yes, absolutely.